script/build-external-libs/fetch-wownero.sh

```diff
@@ -10,10 +10,10 @@ cd $EXTERNAL_LIBS_BUILD_ROOT
 
 
 # url="$base_dir/vendor/wownero"
-url="https://github.com/wownero/wownero"
-# url="https://github.com/wowario/wownero"
-version="v0.5.0.2"
-# version="1b5cefa"
+# url="https://github.com/wownero/wownero"
+url="https://github.com/wowario/wownero"
+# version="v0.5.1.0"
```

Comment:
v0.6.0.0

Reply:
Will use the tagged version in a later update